src/main/java/org/spongepowered/api/item/inventory/entity/UserInventory.java

```diff
@@ -32,7 +32,7 @@
 
 /**
  * Represents the inventory of a {@link User}.
- * It consists of a {@link MainPlayerInventory} (containing the main {@link GridInventory}
+ * It consists of a {@link PrimaryPlayerInventory} (containing the main {@link GridInventory}
  * and the {@link Hotbar}) and an {@link EquipmentInventory}
  */
 public interface UserInventory<T extends User> extends CarriedInventory<T> {
```

Comment:
Thinking about the armor and equipment inventory, what would be returned when querying for a `EquipmentInventory` class?

Reply:
It should detect that all armor slots are contained in the other. 
So you should get one EquipmentInventory (the bigger one) back